modules/driver-mongo/src/test/kotlin/datamaintain/db/driver/mongo/test/AbstractMongoDbTest.kt

```diff
@@ -2,19 +2,20 @@ package datamaintain.db.driver.mongo.test
 
 import datamaintain.core.script.ScriptWithoutContent
 import datamaintain.db.driver.mongo.MongoDriver
+import com.mongodb.client.MongoClients
+import org.bson.Document
 import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.BeforeEach
-import org.litote.kmongo.KMongo
 
 abstract class AbstractMongoDbTest {
     val databaseName = "datamaintain-test"
-    val mongoUri = "localhost:27017"
+    val mongoUri = "localhost:27018"
```

Comment:
Currently not a real mongoURI (mongoURI start with mongodb://).
Should I :
- Rename mongoUri to mongoHost
- Consider mongoURI to be a real mongoURI
- ...

Reply:
It would be dangerous to consider mongo URI to be a real mongo URI. Forgetting ```mongodb://``` or misspelling it is easy and could lead to bugs difficult to identify. Thus "mongoUri" is a bad name. "mongoHost" sounds better.